### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by Kushal997-das** [Kushal997-das/Project-Guidance#152] [Q&A] [answered]
What if we share this project with everyone so that they can able to learn something right ?

**Comment by Programmer1473** [comment]
@Kushal997-das  Yes this would be great :)

**Reply by Kushal997-das** [reply]
Yes @Programmer1473

**Comment by Kushal997-das** [comment, marked as answer]
👍

**Comment by UG-SEP** [comment]
@Kushal997-das,
If you know web development then you can create a website which contains all the project there with a brief description and option to download source code, download .exe file and shows Github repo

**Reply by Priyanshu-Singhz** [reply]
Yes , that could also be done !
